eureka-cli/cmd/deployUi.go

```diff
@@ -61,13 +63,23 @@ func DeployUi() {
 		}
 
 		slog.Info(deployUiCommand, "### UPDATING KEYCLOAK PUBLIC CLIENT", "")
+		masterAccessToken := internal.GetKeycloakMasterRealmAccessToken(createUsersCommand, enableDebug)
 		internal.UpdateKeycloakPublicClientParams(deployUiCommand, enableDebug, tenant, masterAccessToken)
 
 		slog.Info(deployUiCommand, "### COPYING PLATFORM COMPLETE UI CONFIGS ###", "")
-		internal.RunCommandFromDir(deployUiCommand, exec.Command("cp", "-R", "-f", "eureka-tpl/*", "."), outputDir)
-
-		slog.Info(deployUiCommand, "### PREPARING PLATFORM COMPLETE UI CONFIGS ###", "")
-		internal.PrepareStripesConfigJson(deployUiCommand, outputDir, tenant)
+		files, err := filepath.Glob("eureka-tpl/*")
+		if err != nil {
+			// Handle error if the pattern doesn't match any files
+			slog.Info("Failed to glob files: %v", err)
+		}
+		if len(files) == 0 {
+			slog.Info("No files matched in eureka-tpl/*. Not copying.")
```

Comment:
Also this `eureka-tpl` path doesn't seem to exist in the `R1-2024` branch of platform-complete. Not sure why that is but we probably don't want this code to break because of that. The UI seems to build without this directory.

Reply:
Yep, this only exists in the `snapshot` branch at the moment: 
![image](https://github.com/user-attachments/assets/887e6059-25b5-4806-a1fe-10f36d005305)

Q (R1-2024-csp-5) and even the `master` branch do not have this folder. It's probably best to just avoiding building the UI on the older versions, as they are adapted more for Okapi workflow.